frontend/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "cryoet-data-portal",
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "build": "pnpm --parallel build",
+    "clean": "rm -rf .cache && pnpm --parallel clean",
+    "clean:node-modules": "rm -rf node_modules && rm -rf packages/*/node_modules",
+    "dev": "pnpm --parallel dev",
+    "dev:data-portal": "pnpm --filter data-portal dev",
+    "dev:graphiql": "pnpm --filter graphiql dev",
+    "lint": "run-p lint:*",
+    "lint:packages": "pnpm --parallel lint",
```

Comment:
what does pnpm lint for exactly? I want to add prettier to our [pre-commit config](https://pre-commit.com/) but it looks like I would have to do [hacky things for pnpm stuff](https://stackoverflow.com/questions/76268277/pnpm-for-pre-commit-custom-hook)